common/preprocessor.f90

```diff
@@ -15,14 +15,17 @@
 !
 !--------10--------20--------30--------40--------50--------60--------70--------80--------90--------100-------110-------120--------130
 subroutine print_optimize_message
+  use Global_Variables, only: cfunction
   implicit none
   print *, 'Preprocessor: '
-#ifdef ARTED_SC
-  print *, '  ARTED_SC'
-#endif
-#ifdef ARTED_MS
-  print *, '  ARTED_MS'
-#endif
+
+select case(cfunction)
+case ("arted_sc")
+  print *, ' arted_sc'
+case ("arted_ms")
```

Comment:
@yhirokawa san, could you also review this patch? I have confirmed that this modification does not change the physical results. But, I'm not so confident about the performance analyzer.

@uemoto1 san, thank you for quick implementation! I have confirmed that this patch does not change the physical results! It's ready for merge for me. But, I have a couple of comments. If you think they are feasible, please modify the code.

1. I personally think we should remove the prefix of "arted_" from the introduced variable, "cfunction" since it's obvious in the ARTED input. Furthermore, we should not use "arted" as a control variable even for SALMON since it's not ARTED but a specific mode of SALMON.

2. The control variable "cfunction" should be replaced by a more meaningful variable name, e.g. "calc_mode", since it's difficult to understand its meaning.